src/wallet/rpcdump.cpp

```diff
@@ -62,7 +62,7 @@ std::string DecodeDumpString(const std::string &str) {
     for (unsigned int pos = 0; pos < str.length(); pos++) {
         unsigned char c = str[pos];
         if (c == '%' && pos+2 < str.length()) {
-            c = (((str[pos+1]>>6)*9+((str[pos+1]-'0')&15)) << 4) | 
+            c = (((str[pos+1]>>6)*9+((str[pos+1]-'0')&15)) << 4) |
```

Comment:
Unrelated whitespace change.

Reply:
I mentioned that in the PR, is it an issue?

Reply:
I isn't really a problem, but you should try to avoid making such changes. Atom can be set to not make any whitespace changes at all.

Reply:
It is a problem, as it often breaks backporting cherry-picks.

Reply:
Fixed.